wrangler-transform/src/main/java/io/cdap/wrangler/mock/SQLDialect.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.wrangler.mock;
+
+/**
+ * Mock implementation for SQLDialect.
+ */
+public enum SQLDialect {
```

Comment:
This should come from the platform.

Reply:
Resolving this for now as this is not relevant for ANSI SQL conditions.